tracing/README.md

```diff
@@ -0,0 +1,9 @@
+## Tracing  
+- [AsyncWrap](./AsyncWrap/README.md) - Most native objects are represented indirectly by the [AsyncWrap class](https://github.com/nodejs/node/blob/master/src/async-wrap.h), so hooks have been added to that class to trace lifecycle (init/destroy) and callback activity (pre/post) related to these objects.
+
+- [OS Tracing](./os-tracing/README.md) - LTTng (Linux), DTrace (Linux, OSX), ETW (Windows)
```

Comment:
Also SystemTap (Linux).  The dtrace probes don't work on Linux, AFAIK (maybe in Oracle Linux, but no one uses that.)